PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+[//]: # (Note: See http://habitica.wikia.com/wiki/Using_Habitica_Git#Pull_Request for more info)
+
+[//]: # (Put Issue # here, if applicable)
+Fixes #______
```

Comment:
Good point. I did it this way because it will send the reference to the
issue, but it also is what the direction is on the wiki. Do you think there
should be a line underneath that asks for the issue's URL?

On Sat, Mar 19, 2016 at 6:59 PM Alys <notifications@github.com> wrote:

> In PULL_REQUEST_TEMPLATE.md
> <https://github.com/HabitRPG/habitrpg/pull/6928#discussion_r56757127>:
>
> > @@ -0,0 +1,13 @@
> > +[//]: # (Note: See http://habitica.wikia.com/wiki/Using_Habitica_Git#Pull_Request for more info)
> > +
> > +[//]: # (Put Issue # here, if applicable)
> > +Fixes #______
>
> Can this be an instruction to copy the issue's URL? That removes the
> chance of people mistyping the issue number.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/HabitRPG/habitrpg/pull/6928/files/cf8b232d9498abed41c6d5b98dbb26ce4bf68f52#r56757127>
>